### ISSUE TITLE
macOS: can't get through main menu (cursor is misaligned?)

### ISSUE THREAD

**Issue by Janiczek** [alexbatalov/fallout2-ce#160]
Hello!
First of all thank you for this project!

I'm trying to play this on a macOS 12.6 (M1), fallout2-ce [v.1.1.0](https://github.com/alexbatalov/fallout2-ce/releases/tag/v.1.1.0).

With no settings (clean install, no cfg files), when the game opens in fullscreen (on a Retina screen):
* I can move the cursor but clicking the main menu buttons does nothing.
* Also, clicking the mouse to skip the intro movies doesn't work, I have to do that with Spacebar.

See the video below. In it I'm trying to click the buttons but ultimately have to exit the game with Esc.

https://user-images.githubusercontent.com/149425/193066477-cff565ca-26bb-402c-905e-53fbf300995e.mp4

----

This problem doesn't exist when I create `f2_res.ini` with
```ini
[MAIN]
WINDOWED=1
```

With that, mouse clicks do work normally:

https://user-images.githubusercontent.com/149425/193067045-d4a1004f-1a32-4365-b5b8-0169e20a285f.mp4

**Comment by alexbatalov** [comment]
Hey, I've seen this a couple of times but cannot reliably reproduce it. I plan to get rid of relative mouse mode, which will likely fix this problem as well.

**Comment by ckrokosh** [comment]
I'm on an intel iMac running Monterey and have the same issue. As a temporary workaround to get fullscreen working try starting it using the spotlight shortcut (command-space). I find if I leave my mouse near the center of the screen and launch it through spotlight everything works fine. I use Alfred to launch apps vice spotlight and that works too. If I launch it by double clicking on the app icon it doesn't work, same if I click on the spotlight icon and launch it that way.  It also seems to launch fine from the recent items in the apple menu.

**Comment by alexbatalov** [comment]
Still cannot reproduce on MBP 13" and 16". I've tried moving cursor on all edges before running the game. Tried double clicking, opening with Command-O, running from Dock and Launchpad. In some tests (mouse cursor set to screen edge, skipping movies with keyboard) cursor jumps from initial center position in an unspecified direction (usually to top), then moves as usual. Even in this scenario clicking always work. Again, I've seen this problem myself a couple of times, but cannot reliably reproduce it. Please describe what you're doing to with more details and possibly a video.

**Comment by ckrokosh** [comment]
So I think I may have found the cause. I have a second monitor attached to my iMac. I disconnected it and now fallout ce works correctly no matter which way I try and launch it.

**Comment by Janiczek** [comment]
Agreed - I have multiple displays as well, so this might be the necessary condition for the bug to show up.

**Comment by Yerk88** [comment]
Where can i find f2_res.ini file on mac os?
